jobs/servicebroker/templates/config.json.erb

```diff
@@ -29,11 +29,11 @@
         },
     },
     'httpRequestTimeout' => p('service_broker.http_request_timeout'),
-    'tls' => {
-        'keyFile' => "/var/vcap/jobs/servicebroker/config/certs/servicebroker/server.key",
-        'certFile' => "/var/vcap/jobs/servicebroker/config/certs/servicebroker/server.crt",
-        'caCertFile' => "/var/vcap/jobs/servicebroker/config/certs/servicebroker/ca.crt",
-    },
+    # 'tls' => {
```

Comment:
We should make tls optional instead of removing it, so if the manifest contains tls then it should go to configuration.

Reply:
@rohitsharma04  After broker registered to gorouter which is a http router, tls is useless and will cause route error.